src/components/bars/SSINavigationBar/index.tsx

```diff
@@ -43,45 +43,47 @@ class SSINavigationBar extends PureComponent<BottomTabBarProps, IState> {
 
   render() {
     return !this.state.keyboardVisible ? (
-      <SafeAreaView edges={['bottom']}>
-        <Container>
-          {this.props.state.routes.map((route, index: number) => {
-            const {options} = this.props.descriptors[route.key];
-            const isFocused = this.props.state.index === index;
+        <View style={{backgroundColor: backgrounds.primaryDark}}>
```

Comment:
if we go for this option, would like to move this to the styling file of the SSINavigationBar